app/api/parties_percentage_api.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class PartiesPercentageApi < Grape::API
```

Comment:
As we'll have some other similar endpoints to do, we might name it `AggreationsApi` in order to have them coupled together. Please let me know what do you think?